### ISSUE TITLE
[BUG] [a11y] With NVDA, space bar does not select row in ui-grid

### ISSUE THREAD

**Issue by Sonakshi14** [angular-ui/ui-grid#7284]
**Describe the bug**
With NVDA, space bar does not select row in ui-grid

**To Reproduce**
1. Go to http://ui-grid.info/docs/#!/tutorial/Tutorial:%20210%20Selection
2. Focus on checkbox, press Space, row selection is successful
3. Try above with NVDA 
4. Row selection with Space does not work with NVDA

**Desktop (please complete the following information):**
 - Browser: Mozilla with NVDA

**Comment by orneryd** [comment]
Thanks for contributing! This repository has been transferred, and we're preparing the new 5.x release. We're closing older issues and pull requests as part of the reset. Please join us on Discord to follow the work or continue the conversation: https://discord.gg/Baz4w8ZWN
